docs/assets/CenteredImage.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+/**
+ * Component for centering an image in the docs
+ *
+ * @param {*} props :
+ *  src: image source
+ *  alt: image alternative text
+ *  imgStyle: object, any image styles (props can be passed in as: `imgStyle={{ maxHeight: "600px", border: "solid" }}`)
+ * @returns a component for uniformally centering images in the doc
+ */
+export default function CenteredImage({ src, alt, ...delegated }) {
+  let imgStyleString;
+  if (
+    delegated.imgStyle != undefined &&
+    Object.keys(delegated.imgStyle).length === 0
+  ) {
+    imgStyleString = {};
+  } else {
+    imgStyleString = delegated.imgStyle;
+  }
+
+  return (
+    <div style={{ textAlign: "center" }}>
+      <img src={src} alt={alt} style={imgStyleString} />
+    </div>
+  );
```

Comment:
If you destructure the delegated prop object into the image it will handle all of the style props for you!
```suggestion
  return (
    <div style={{ textAlign: "center" }}>
      <img src={src} alt={alt} {...delegated} />
    </div>
  );
```

Reply:
This means you can delete lines 13-21